components/developer/cmake/test/results-i386.master

```diff
@@ -1012,9 +1056,11 @@ Total Test time (real) = SEC sec
 The following tests FAILED:
 	 39 - FortranOnly (Failed)
 	 40 - VSGNUFortran (Failed)
-	 83 - Module.CheckIPOSupported-Fortran (Failed)
-	237 - Fortran (Failed)
-	238 - FortranModules (Failed)
-	267 - CMakeOnly.CompilerIdFortran (Failed)
-	434 - RunCMake.CompilerLauncher (Failed)
+	 68 - CompileOptions (Failed)
+	 85 - Module.CheckIPOSupported-Fortran (Failed)
+	257 - Fortran (Failed)
+	258 - FortranModules (Failed)
+	292 - CMakeOnly.CompilerIdFortran (Failed)
+	348 - RunCMake.FileAPI (Failed)
```

Comment:
There are two new test failures, have you investigated them?

Reply:
I have checked them, the CompileOptions failure is due to selecting sunstudio in the test, the FileAPI test also defaults to sunstudio... maybe I should just patch to avoid using /usr/bin/cc by default. These symlinks in /usr are annoying...